pom.xml

```diff
@@ -85,9 +85,21 @@
     <dependencies>
         <dependency>
             <groupId>com.exasol</groupId>
-            <artifactId>virtual-schema-common-java</artifactId>
-            <version>${vscommon.version}</version>
+            <artifactId>virtual-schema-common-document</artifactId>
+            <version>${vs-common-document.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>virtual-schema-common-document</artifactId>
+            <version>${vs-common-document.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>dynamodb</artifactId>
```

Comment:
This does not require a version?

Reply:
no, the version is set for all AWS packages here https://github.com/exasol/dynamodb-virtual-schema/blob/cd028f74dfa1a83a28196609e1b76c9ba955ac7e/pom.xml#L54

Reply:
:+1: